src/pages/status/migration/styles.module.css

```
@@ -48,81 +47,29 @@
 .migration_details_filter_button:hover {
   text-decoration: underline;
 }
-/*
-  Filter buttons in order:
-  { Done, In PR, Awaiting PR, Awaiting parents, Not solvable, Bot error }
-*/
 
 .migration_details_filter_done {
-  background-color: rgb(60, 1, 74);
-  border: 3px solid rgb(60, 1, 74, 0.4);
-  color: var(--ifm-color-secondary);
-}
-
-.migration_details_filter_done_on {
-  background-color: rgb(60, 1, 74, 0.4);
-  border: 3px solid rgb(60, 1, 74);
-  color: var(--ifm-color-primary-darkest);
+  background-color: var(--ifm-color-success);
 }
 
 .migration_details_filter_in_pr {
-  background-color: rgb(43, 93, 131);
-  border: 3px solid rgb(43, 93, 131, 0.4);
-  color: var(--ifm-color-secondary);
-}
-
-.migration_details_filter_in_pr_on {
-  background-color: rgb(43, 93, 131, 0.4);
-  border: 3px solid rgb(43, 93, 131);
-  color: var(--ifm-color-primary-dark);
+  background-color: var(--ifm-color-gray-500);
```

```suggestion
  background-color: var(--ifm-color-emphasis-500);
```

This one is the same but reacts to dark mode to maintain contrast.